### ISSUE TITLE
(user profiles) Create the admin user page that displays all the users

### ISSUE THREAD

**Issue by AndrewM131** [uchicago-cs/chigame#32]

**Comment by AndrewM131** [comment]
Description: We want to create an admin page that is only accessible to staff_users. This page will display all users' usernames and emails for the admin to view. 
Our current progress: We created our first pull request.

**Comment by AndrewM131** [comment]
Status update: After the first pull request, our team member requested changes to correct our path in urls.py for users_list. Also, he asked us to add in a login in the views.py in order to access the user_list. After fixing this issue, we asked for a review and were asked to resolve an error in the template not being created. this was resolved by fixing the views.py to include users/user_list.html instead of user_list. Once approved for a PR, we had to resolve two merge conflicts in views.py and urls.py.

**Comment by AndrewM131** [comment]
Closing comment: We completed this task by implementing an "if staff_user statement" in user_detail.html, and then displays the users. 
Files edited: users/urls.py, users/views.py, templates/users/user_detail.html
This task was completed by @ishiii-s and @AndrewM131. We met on Monday, Oct 30th, 2023 where we worked together in coding on my laptop, and researching and referencing code on Ishi's laptop. We created the first pull request, and I fixed any minor changes in the code asked to be changed by the reviewers.
